### ISSUE TITLE
Error while importing the BossRoom project in Unity 2020.3

### ISSUE THREAD

**Issue by AbhimanyuBundela** [Unity-Technologies/com.unity.multiplayer.samples.coop#265]
**Describe the bug**

**To Reproduce**
1. Extract the provided zip file.
2. Add project in the Unity (my version Unity 2020.3)
3. Wait while project is importing.

**Expected behavior**
A BossRoom Project should be opened.

**Screenshots**
![Screenshot (6)](https://user-images.githubusercontent.com/19886057/114080850-7abd3200-98c9-11eb-8e45-fcdae2205ddf.png)
![Screenshot (7)](https://user-images.githubusercontent.com/19886057/114089369-b0ffaf00-98d3-11eb-807b-dc09f3a4db12.png)

**Environment (please complete the following information):**
 - Windows 10
 - Unity Version: 2020.3
 - BossRoom Version: v0.1.0

**Additional context**
- I already have the Git and Git LTS installed.

**Comment by JafetJuan** [comment]
I had the same problem, at least at the beginning. At the end i have my project full working.
The steps that I followed are the following:
- Make sure you have the latest version of git (for me (2.31.1) If not, reinstall Git.
- Check the UnityHub versión, update and fully close it.
- Init UnityHub and start the project. Wait for packet install and you´re done.

Is important to close UnityHub for detect Git on your system to get the new packages.
Hope it solves this problem as I solved it and works well for you.
